spec/README.md

```diff
@@ -79,12 +79,16 @@ A reference to a _term_ looks like this.
 > The provisions of the stability policy are not in effect until
 > the conclusion of the technical preview and adoption of this specification.
 
-Updates to this specification will not change
-the syntactical meaning, the runtime output, or other behaviour
-of valid messages written for earlier versions of this specification
-that only use functions defined in this specification.
-Updates to this specification will not remove any syntax provided in this version.
-Future versions MAY add additional structure or meaning to existing syntax.
+Updates to this specification shall not make any well-formed or valid _message_ invalid.
+Updates to this specification shall not remove any syntax provided in this version.
```

Comment:
These two promises are already covered by
> Updates to this specification will not introduce message syntax that, when parsed according to earlier versions of this specification, would produce syntax or data model errors.

further down in the policy.

Also, the second one follows from the first.

Reply:
Eh, I confused myself. The text further down is promising things the other way than what's proposed here. Leaving this comment in, though, as these two parts should probably be merged into one.